### PR TITLE
docs: Add GridColumn docs

### PR DESCRIPTION
**Pull request by damassi**
This adds docs to the new `GridColumn` component. 

Screen Shot 2020-10-23 at 8 05 14 PM

### CODE REVIEW

File: packages/palette-docs/content/docs/elements/layout/GridColumn.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+name: GridColumn
+---
+
+(WIP text) The Grid element is a layout container for rendering a Grid
```
Comment (damassi) [inline packages/palette-docs/content/docs/elements/layout/GridColumn.mdx:5]: @dzucconi - would you mind pushing a commit that updates this text?

File: packages/palette-docs/content/docs/elements/layout/GridColumn.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Examples
+
+<Playground>
+  <GridColumns border="1px solid blue" position="relative">
+    <Column border="1px solid red" span={4}>
+      <Text>Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto deleniti recusandae ullam laudantium ut reiciendis, doloribus qui sequi id ea, ad suscipit eos placeat magnam consequatur sunt, quaerat eius saepe.</Text>
```
Comment (damassi) [inline packages/palette-docs/content/docs/elements/layout/GridColumn.mdx:12]: Not fun wrangling mdx 😦

File: packages/palette-docs/content/docs/elements/layout/GridColumn.mdx (hunk 1 above)
Comment (dzucconi) [reply packages/palette-docs/content/docs/elements/layout/GridColumn.mdx:5]: 👍

**Comment by dzucconi** [comment]
Ah, woops, shouldn't have pushed a version bump.

**Comment by artsyit** [comment]
:rocket: PR was released in [`v13.14.0`](https://github.com/artsy/palette/releases/tag/v13.14.0) :rocket:

**Pull request merged**